Задания с 18.02.2022/Task1/Task1/BubbleSort.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
Надо ещё свой неймспейс всегда объявлять:
```suggestion
namespace BubbleSort;

﻿using System;
```